src/widget.js

```diff
@@ -1,8 +1,12 @@
+
```

Comment:
Sorry, i am confused, what's the relationship between src/widget.js and _ts/widget.ts?